clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -86,31 +135,29 @@ class DockerRegistryClient {
     Response response
     try {
       if (token) {
-        response = registryService.getTags(repository, token)
+        response = withToken(token)
       } else {
-        response = registryService.getTags(repository)
+        response = withoutToken()
       }
     } catch (RetrofitError error) {
       if (error.response.status == 401) {
-        dockerToken = tokenService.getToken(repository, error.response.headers)
+        dockerToken = tokenService.getToken(target, error.response.headers)
         token = "Bearer ${dockerToken.bearer_token ?: dockerToken.token}"
-        response = registryService.getTags(repository, token)
+        response = withToken(token)
       } else {
         throw error
       }
     }
 
-    return (DockerRegistryTags) converter.fromBody(response.body, DockerRegistryTags)
+    response
```

Comment:
Please use `return` here to be consistent with the original methods in this class.